admin/class-boldgrid-backup-admin-jobs.php

```diff
@@ -87,6 +87,24 @@ public function add( $args ) {
 		$this->set_jobs();
 		$this->jobs[] = $args;
 		$this->save_jobs();
+
+		$this->core->logger->init( 'jobs-queue' );
```

Comment:
If you init the `jobs-queue` here, then anywhere else in the code that init `core->logger` will begin writing to that file.

For example, if you had a backup process running, the `core->logger` will be writing to the log file for that backup. Here, you just changed which file it will be writing too... Need to have distinct loggers.

Suggest creating a new property or variable `$jobs_logger`

Reply:
That makes sense. Didn't think of that. Still a little new to the Logger used here. Will do